tyr-runner/src/test/running_teamcity_test_instance.adoc

```diff
@@ -1,22 +1,60 @@
-1. get TeamCity e.g. `wget https://www.jetbrains.com/teamcity/download/download-thanks.html?platform=linux`
-2. in TeamCity/bin run `./runAll.sh start`
-3. you may enter TeamCity UI with localhost:8111 taped in your browser
-4. create e.g. Test Repo project with Root Project as parent project
-5. create Git VCS root attached to your Github repository e.g. test-repo
-    - Type of VCS: Git
-    - Default branch: refs/heads/master
-    - Branch specification: +:refs/*/head
-6. create Build Configuration e.g. Name: Build, Build configuration ID: TestRepo_Build in General Settings of your created project (e.g. Test Repo)
-7. attach VCS Root (same settings as before)
-8. add Maven Build Step
-    - Execute step: If all previous steps finished successfully
-    - Goals: clean test
-    - Path to POM file: pom.xml
-    - Additional Maven command line parameters: -Dmaven.test.failure.ignore=true
-9. create tyr/tyr-runner/tyr_config_dir
-10. Inside this directory create:
-    - admin-list.txt (add your TeamCity user name)
-    - user-list.txt (add your TeamCity user name)
-    - format.yaml
-    - config.properties
-11. run `mvn clean compile quarkus:dev -Dtyr.config.dir="/yourPath/tyr/tyr-runner/tyr_config_dir"`
\ No newline at end of file
+Here we set Teamcity so it contains 2 build steps: one successful and other one failing because of system property `fail-build` set in build steps.
+These build steps should be by VCS root attached to GitHub project (described below), which contains test system property `fail-build`.
+
+1. download TeamCity for Linux from https://www.jetbrains.com/teamcity/download/ .TAR.GZ -> Linux/macOS(.tar.gz)
+2. extract files with `tar xvzf TeamCity-2019.1.2.tar.gz`
+3. run `cd TeamCity/bin/`
+4. run `./runAll.sh start`
+5. access TeamCity web UI via web browser with address http://localhost:8111
+    - set Data Directory location on the TeamCity server machine (you can use default)
+    - select the database type (use Internal)
+    - accept license agreement
+    - create administration account (e.g. username=admin, password=123456)
+6. create project (Administration -> Projects -> +Create project)
+    - Manually E.g.:
```

Comment:
so Administration -> Projects -> +Create project and then I need to manually fill in these values?

maybe just drop this line as it's a little confusing.